python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -501,6 +501,12 @@ def vertices_ids(self):
         """
         return self.get_vertices()
 
+    def vertex_types_from_numerals(self, nums):
+        return self.__vertex_prop_dataframe[self.type_col_name].dtype.categories[nums]
+
+    def edge_types_from_numerals(self, nums):
+        return self.__edge_prop_dataframe[self.type_col_name].dtype.categories[nums]
+
```

Comment:
These should probably have docstrings since they're in the public API. Also, can you put `FIXME`s in the corresponding test files for these (or better yet, disabled empty tests).  Same for MG.

Reply:
I did even better and wrote the full tests for those functions.

Reply:
and docstrings, obviously